x-pack/legacy/plugins/siem/public/containers/anomalies/anomalies_query_tab_body/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { hostsModel, networkModel } from '../../../store';
+import { ESTermQuery } from '../../../../common/typed_json';
+import { NarrowDateRange, AnomaliesNetworkTableProps } from '../../../components/ml/types';
+import { UpdateDateRange } from '../../../components/charts/common';
+import { SetQuery } from '../../../pages/hosts/navigation/types';
+
+interface QueryTabBodyProps {
+  type: hostsModel.HostsType | networkModel.NetworkType;
+  filterQuery?: string | ESTermQuery;
+}
+
+export type AnomaliesTableComponent = Omit<AnomaliesNetworkTableProps, 'type'> & {
```

Comment:
The name of this `type` does actually sounds like a component here, and at the same time we do also use the same name as a props/component, therefore I'd recommend to rename this type.